### DISCUSSION TITLE
How to debug lf?

### DISCUSSION THREAD

**Discussion by samsze0** [gokcehan/lf#1496] [Q&A] [answered]
I want to see the log outputs of my custom scripts but I can't find any mentions on how to debug `lf` in general

**Comment by joelim-work** [comment, marked as answer]
Use `-log` option when running `lf`, for example `lf -log /path/to/logfile`. If you want to test shell scripts, then you will have to explicitly write to the log file, like `echo test >> /path/to/logfile`.
